website_and_docs/content/documentation/grid/configuration/toml_options.en.md

```diff
@@ -188,4 +188,44 @@ driver.get("https://selenium.dev");
 driver.quit();
 ```
 
+### Retrieving downloaded files from Node.
 
+To be able to retrieve the files that were downloaded by a test at the Node, its location can be specified as below:
+
+```toml
+[node]
+downloads-path = "/usr/downloads"
+```
+
+Here is a Java example showing how to retrieve the file from Node.
+
+```java
+// The file can be downloaded by accessing
+// curl -X GET "http://localhost:4444/session/<sessionId>/se/file?filename=my_file.pdf"
+
+HttpRequest request = new HttpRequest(
+    HttpMethod.GET,
+    String.format("/session/%s/se/file", driver.getSessionId()));
+request.addQueryParameter("filename", "my_appointments-1.pdf");
+// Here gridUrl is http://localhost:4444
+try (HttpClient client = HttpClient.Factory.createDefault().createClient(gridUrl)) {
+  HttpResponse response = client.execute(request);
+  Map<String, Object> map = new Json().toType(string(response), Json.MAP_TYPE);
+  // The returned map would contain 2 keys viz.,
+  // filename - This represents the name of the file (same as what was provided by the test)
+  // contents - Base64 encoded String which contains the zipped file.
+  String encodedContents = map.get("contents").toString();
+
+  //The file contents would always be a zip file and has to be unzipped.
+  Zip.unzip(encodedContents, dirToCopyTo);
+}
+```
+
+##### Points to remember:
+
+* The endpoint to `GET` from is `/session/<sessionId>/se/file?filename=`
+* The response contains two keys viz.,
+    * `filename` - Same as what was specified in the request.
+    * `contents` - Base64 encoded zipped contents of the file.
+* The file contents are Base64 encoded.
+* The contents need to be unzipped.
```

Comment:
This is duplicating the information because the contents are almost the same as the `cli` section. Maybe we should just show the `toml` part and link to the `cli` section for the detailed explanation.